confluence-xml/src/main/java/org/xwiki/contrib/confluence/filter/input/ConfluenceXMLPackage.java

```diff
@@ -1125,6 +1127,27 @@ private void readBlogPostObject(XMLStreamReader xmlReader)
         }
     }
 
+    private void readLabellingObject(XMLStreamReader xmlReader)
+        throws XMLStreamException, FilterException, ConfigurationException
+    {
+        ConfluenceProperties properties = new ConfluenceProperties();
+
+        long labellingId = readObjectProperties(xmlReader, properties);
+        saveObjectProperties(properties, labellingId);
+
+        // Since confluence 8.0, the labellings are not part of the Page Object anymore.
+        Long pageId = properties.getLong("content", null);
+
+        if (pageId != null) {
+            ConfluenceProperties fileProperties = getPageProperties(pageId, true);
```

Comment:
Is there a specific reason to call the variable "fileProperties" and not something like "pageProperties" as the call would suggest ?

Reply:
nope. renaming as you suggested